_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1386,9 +1386,15 @@ cloudtrail_s3_bucket_name = "<CLOUDTRAIL_BUCKET_NAME>"
 config_s3_bucket_name     = "<CONFIG_BUCKET_NAME>"
 ----
 
-Take note of all of this data, as you'll need it again shortly!
+If you followed the http://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#prepare-your-infrastructure-live-repository[steps for preparing your `infrastructure-live` repo],
+now you can update the `accounts.json` file with the account IDs from the Terraform output! Also `account.hcl` files
```

Comment:
Why do we need `account.hcl` files?

Reply:
+1. Does the second sentence mean we also need to update the `account.hcl` files? I can see the account.hcl in the RefArch, but is that something we want them to create too when they're using the service catalog too?

Reply:
Are you suggesting that we only need `accounts.json`, or am I missing the point of the questions?

Reply:
Yea, it seems like `accounts.json` is all we'd need... What's in `account.hcl`?

Reply:
I've updated this to allow for both versions - whichever the customers are wanting, and I've added the code examples for that.